### DISCUSSION TITLE
Message: Specified argument was out of the range of valid values. Parameter name: restrictedAccessControlGroups

### DISCUSSION THREAD

**Discussion by mohammadamer** [pnp/pnpcore#1190] [General] [answered]
I have issue with site.UpdateAsync(); I got this error 

```
System.Private.CoreLib: Result: Failure
Exception: System.AggregateException: One or more errors occurred. (CSOM service exception)
 ---> HttpResponseCode: 200
Message: Specified argument was out of the range of valid values.
Parameter name: restrictedAccessControlGroups
ClientRequestId: 78a6b6a0-800f-6000-ad62-f9f3d247aa6c
ErrorTypeName: "System.ArgumentOutOfRangeException"
```

I didn't have this issue few weeks ago. I'm trying to change DenyAddAndCustomizePages 

```
  var site = await Ctx.GetSiteCollectionManager().GetSiteCollectionPropertiesAsync(Ctx.Uri);
  site.DenyAddAndCustomizePages = status;
  await site.UpdateAsync();
```

Has anyone face such issue before when updating a site ?

**Comment by jansenbe** [comment]
@mohammadamer : I actually faced that yesterday as well and fixed this, if you pick the latest nightly things should work again

**Comment by mohammadamer** [comment]
@jansenbe 
Thanks for the updates. 
In our environment we are always use PnP Core released versions so we might wait a bit for the release if it will be soon.
May I know when will be the next release?

**Reply by jansenbe** [reply]
@mohammadamer : we've not planned an official release for the coming weeks, would recommend trying the nightly and checking if that at least fixes the problem for you?

**Comment by FlorianLabranche** [comment, marked as answer]
Hello,
I've just faced the same issue.
And I can confirm it is fixed with nightly version of PnP.Core.

**Reply by mohammadamer** [reply]
How can I know the current nightly build ?

**Reply by FlorianLabranche** [reply]
Check the homepage of this github project or nuget.
![image](https://github.com/pnp/pnpcore/assets/23434632/a65a2f39-3f97-4968-a12a-881eb3b5838f)
